app/WebParsing/ParseAll.hs

```diff
@@ -1,11 +1,14 @@
 module WebParsing.ParseAll
-     (parseAll) where
+     (parseCalendar, parseTimetable) where
```

Comment:
Now that we're splitting things up, this file is no longer necessary. Move the definition of `parseCalendar` into `ArtSciParser`, and rename `getAllCourses` to `parseTimetable`. Then change the codebase to import from those modules rather than `ParseAll`.